frontend/packages/rhoas-plugin/src/catalog/catalog-content.ts

```diff
@@ -16,4 +12,14 @@ export const RHOASService = {
   icon: kafkaIcon,
   ctaLabel: 'Connect',
   details: 'tbd',
-};
+  model: {
```

Comment:
This model should be used to create resource later. 
It is optional for this PR but adding it here as example how future catalog will look like. 

Dynamic service capabilities will be enabled after we ship 4.8 so the only thing we want is to have well defined catalog content like here - no other changes are needed.